mlflow/tracking/fluent.py

```diff
@@ -196,7 +227,35 @@ def start_run(run_id=None, experiment_id=None, run_name=None, nested=False):
 
 
 def end_run(status=RunStatus.to_string(RunStatus.FINISHED)):
-    """End an active MLflow run (if there is one)."""
+    """End an active MLflow run (if there is one).
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Start run and get status
+        mlflow.start_run()
+        run = mlflow.active_run()
+        print("run_id: {}; status: {}".format(run.info.run_id, run.info.status))
+
+        # End run and get status
+        mlflow.end_run()
+        run = mlflow.get_run(run.info.run_id)
+        print("run_id: {}; status: {}".format(run.info.run_id, run.info.status))
+        print("--")
+
+        # Check for any active runs
+        print("Active runs: {}".format(mlflow.active_run()))
```

Comment:
```suggestion
        print("Active run: {}".format(mlflow.active_run()))
```

`Active runs` souns like there is a  possibility that there are multiple active runs.

Reply:
good point.